mycore-solr/src/main/resources/configset/mycore_classification/conf/managed-schema

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Apart from some differences in the formatting, these appears to be the same configsets that are currently maintained as separate repositories (https://github.com/MyCoRe-Org/mycore_solr_configset_classification and https://github.com/MyCoRe-Org/mycore_solr_configset_main respectively).

I'd have thought that there is a reason, that the configsets live in separate repositories. Is it really indented to change this, or what is the reason for including the configsets here?

Reply:
Yes i like to change that, because the handling with submodules in mir or other projects is bad. You need to always make sure to initialize them and they are always tied to a single commit instead of a branch.

Putting them inside mycore allow overwriting them like all other resources and they are closer to the code (you will have always the right configset for the right mycore version)